### PR TITLE
feat(hogql): database schema page

### PR DESCRIPTION
**Pull request by mariusandra**
## Problem

HogQL is around the corner, yet you don't know what you're querying. 

## Changes

Adds a "Database" view under "data management" that lists all the tables fields available to you via HogQL:

image

It's feature flagged behind a new flag `database`.

## How did you test this code?

- Checked the page visually.
- Created snapshot tests for the rendered database JSON for both PoE on and off.
- I don't know if it's worth a visual regression test right now. It looks like a big regression anyway 😅

### CODE REVIEW

**Comment by mariusandra** [comment]
This is ready for a look. The interface does look quite horrible though, but keep in mind this is behind a flag. I'm happy to do any cosmetic suggestions... and happy to get improvement suggestions in code too. :)

I think this page will become more and more important, especially when we add support for virtual fields (calculated properties?) and other tables.

File: frontend/src/scenes/data-management/database/DatabaseScene.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                                    return <code>{column}</code>
+                                },
+                            },
+                            {
+                                title: 'Type',
+                                key: 'type',
+                                dataIndex: 'type',
+                                render: function RenderType(type) {
+                                    if (type === 'virtual_table') {
+                                        return <LemonTag type="default">Virtual Table</LemonTag>
```
Comment (pauldambra) [inline frontend/src/scenes/data-management/database/DatabaseScene.tsx:55]: not blocking right now but as someone fresh to this page I'm going to wonder "oh, what's a (virtual table|reference|expression)" I'll have an idea as a technical user but we should probably make a link to a section of the docs obvious here in an info button or something

**Review by pauldambra — APPROVED** [review]
LGTM

Makes me want to click on a bunch of the columns and then open that as an insight 🤔

File: frontend/src/scenes/data-management/DataManagementPageTabs.tsx [hunk 2]
```diff
@@ -106,6 +110,16 @@ export function DataManagementPageTabs({ tab }: { tab: DataManagementTab }): JSX
                     key={DataManagementTab.IngestionWarnings}
                 />
             )}
+            {showDatabaseTab && (
+                <Tabs.TabPane
+                    tab={
+                        <span data-attr="data-management-database-tab">
+                            Database <LemonTag type="warning">BETA</LemonTag>
```
Comment (thmsobrmlr) [inline frontend/src/scenes/data-management/DataManagementPageTabs.tsx:117]:
```suggestion
                            Database
                            <LemonTag type="warning" className="uppercase ml-2">
                                Beta
                            </LemonTag>
```

**Review by thmsobrmlr — APPROVED** [review]
Good addition!

I agree that we're going to need some explanations for virtual tables, pdi, poe, etc. once this is going to general availability :)

**Pull request merged**